src/Catalyst.Abstractions/Dfs/BlockExchange/WantedBlock.cs

```diff
@@ -0,0 +1,28 @@
+﻿using System.Collections.Generic;
+using System.Threading.Tasks;
+using Lib.P2P;
+using MultiFormats;
+
+namespace Catalyst.Abstractions.Dfs.BlockExchange
+{
+    /// <summary>
+    ///   A content addressable block that is wanted by a peer.
+    /// </summary>
+    public class WantedBlock
+    {
+        /// <summary>
+        ///   The content ID of the block;
+        /// </summary>
+        public Cid Id;
+
+        /// <summary>
+        ///   The peers that want the block.
+        /// </summary>
+        public List<MultiHash> Peers;
+
+        /// <summary>
+        ///   The consumers that are waiting for the block.
+        /// </summary>
+        public List<TaskCompletionSource<IDataBlock>> Consumers;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Make this field 'private' and encapsulate it in a 'public' property.](https://app.codacy.com/app/catalyst-network/Catalyst.Framework/pullRequest?prid=4612924)